src/shared/logger/loggableType.ts

```diff
@@ -0,0 +1,6 @@
+/*!
+ * Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export type Loggable = Error | string
```

Comment:
I feel like this can be merged into index.ts. That way, you can also dump that file's Loggable type; doesn't make sense to have it duplicated.

Also, add a Typedoc comment for this type to explain what's considered loggable.

Reply:
We'll likely have `Loggable` related methods in this file. Things like "is it an Error?" type functions.

Reply:
(Typedoc comment added)

Reply:
It still doesn't make sense to have two `Loggable` types; I'd still vote to move this there, or at the very least remove the `index.ts` version. If you don't want to merge all additional `Loggable`-related functionality into `index.ts`, maybe create a `loggableUtils.ts` file or something for those functions.